.env.local-example

```diff
@@ -16,3 +16,7 @@ NEXT_PUBLIC_SENTRY_DSN=''
 # Shopify Multipass
 SHOPIFY_MULTIPASS_SECRET=''
 
+# Playwright URL for local runs
+PLAYWRIGHT_TEST_BASE_URL='http://127.0.0.1:3000'
+# Set a product name to use in your own tests
+PLAYWRIGHT_PRODUCT_NAME='Striped Skirt and Top'
```

Comment:
If you add the `PLAYWRIGHT_COLLECTION_HANDLE` var be sure to include it here as well.